app/controllers/events_controller.rb

```diff
@@ -13,11 +13,12 @@ def index
 
   # GET /events/1
   def show
+    @events = nil
```

Comment:
does this do anything? The only place I can find where @events is set is in the index action. When the show action is called, a new instance of the controller will be created and @events is not initialized anymore.